scoring/templates/scoring/home.html

```diff
@@ -16,13 +16,7 @@ <h1 class="mb-3">
             <p class="mb-4" style="max-width: 650px;">Climate Emergency UK assessed all UK councils on the actions they’ve taken towards net zero. The 2025 Action Scorecard assessment consists of 93 questions or less, depending on council type, across 7 different sections, created in consultation with over 90 different organisations and individuals. Each council was marked against these criteria and given a right to reply before the scores underwent a final audit. This work was completed between July 2024 and March 2025.</p>
             <div class="d-flex flex-wrap gap-3">
                 <a class="btn btn-outline-primary" href="{% url 'year_scoring:methodology' plan_year %}"> Learn more about the methodology</a>
-              {% if current_plan_year %}
```

Comment:
@struan do you know whether we still need to keep this `current_plan_year` condition here? I see that, if `current_plan_year` is truthy, we link to `{% url 'scoring:sections' %}` for the Sections, and if it’s falsy, we link to `{% url 'year_scoring:sections' plan_year %}`… 🤔

Reply:
yes, we do. it's to make sure that it links without the year for the current plan year